test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -8080,19 +8079,35 @@ def update_attributes(self, obj_type, bs):
                     self.hooks[id] = Hook(id, binfo, server=self)
                 obj = self.hooks[id]
             elif obj_type == SCHED:
-                if self.scheduler:
-                    self.scheduler.attributes.update(binfo)
+                if id in self.schedulers:
```

Comment:
@vinodchitrali  Can you please explain here what you trying to do? 
As far as what I got is here is you are updating attrs if id is in self.schedulers else create new sched with either sched_host or self.hostname then update attrs for newly created sched.
Now again you are checking id==default then updating self.scheduler why? default id will be there in self.schedulers right?
And Don't we have to take care of diag mode in multi-sched?